CHANGELOG.md

```diff
@@ -1,3 +1,155 @@
+# v1.13.0 (2022-03-15)
+
+## OS Changes
+
+* Set NVMe IO request timeouts for EBS according to AWS recommendations ([#2820])
+* Add `ethtool` to Bottlerocket ([#2829])
+* rpm2img: stripe root filesystem for measureable performance improvement during boot ([#2863]) ([#2879])
```

Comment:
[#2879](https://github.com/bottlerocket-os/bottlerocket/pull/2879) is related to "Label `BOTTLEROCKET-DATA` at runtime"